### PR TITLE
(fix) O3-2694: fixed cancel button caption alignment

### PR DESCRIPTION
**Pull request by elimm**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

## Summary
Minor fix for Cancel button alignment if RTL-based language is selected

## Related Issue
[O3-2694](https://openmrs.atlassian.net/browse/O3-2694)

### CODE REVIEW

File: packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss [hunk 1]
```diff
@@ -180,6 +180,7 @@
 
   & button {
     width: 100%;
-    padding-inline-end: 0;
+    padding-inline: 15px 0;
```
Comment (sherrif10) [inline packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss:183]: LGTM. CC @denniskigen

File: packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss [hunk 2]
```diff
@@ -180,6 +180,7 @@
...
+    padding-block: calc(0.875rem - 3px);
```
Comment (ibacher) [inline packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss:184]: Could you explain this calculation, as in where these numbers come from?

File: packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss (hunk 1 above)
Comment (ibacher) [inline packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss:183]: Please use the Carbon spacing name space for this (this is about `spacing.$spacing-05`; yes we need to clean this file up.

File: packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss (hunk 2 above)
Comment (elimm) [reply packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss:184]: @ibacher It's taken from [here](https://github.com/openmrs/openmrs-esm-core/blob/43a20b2cc5673071ed9c816c20a93682cb9e977e/packages/framework/esm-styleguide/src/_overrides.scss#L385), and seems it also can be simplified to `spacing.$spacing-04`, the result doesn't change.

File: packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss (hunk 1 above)
Comment (elimm) [reply packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss]: done

**Pull request merged**